providers/fcm/src/lib/fcm.provider.spec.ts

```diff
@@ -167,3 +167,78 @@ test('should trigger fcm with apns (ios) override', async () => {
     },
   });
 });
+
+test('should trigger fcm data for both android and ios with override options', async () => {
```

Comment:
We have already an ios test so leave it just for Android and pass only the Android properties, not the APNS ones, please.
In the iOS test, include the APNS `headers` part of the payload to complete the test the other contributor created. 👍🏻

Reply:
Hi @p-fernandez 
Test cases were added, since almost change duplicate #2979 , this PR only contains small change for missing headers (apns) and priority fields (android)

Reply:
What I meant was to separate the iOS and Android testing in separate tests.
```
test('should trigger fcm data for android with override options',
```
and remove from the payload the `apns` object, as we already test it in the test of line 171.

Reply:
The test case was separated for android and ios